examples/MuttsVersionedTrees/MuttsVersionedTrees.jl

```diff
@@ -22,7 +22,7 @@ head = insert!(head, 2);
 
```

Comment:
Oops, this whole file looks like a mistake. If you don't mind, i'm going to just push a commit to this branch to delete this file.

Reply:
Haha yes I should have asked about that!